### ISSUE TITLE
Documentation: Explain "non-reliable" in README.md

### ISSUE THREAD

**Issue by Gdsimms** [danielinux/ttybus#14]
The README.md file currently states "a shared, *non-reliable* bus" and this is scaring off some of my coworkers from adopting this as a solution to a current problem we have. Could you explain somewhere in the documentation what the meaning of "non-reliable" is in this context? Is it just that it can't prevent corruption when multiple devices send simultaneously? If that's all it is, then my other current issue #13 would make the bus reliable by locking out writing on all but one fake_tty.

**Comment by leoheck** [comment]
I was reading the README yesterday and this word also made me uncomfortable. What is the meaning of non-reliable here @Gdsimms ? Can you give us a hint on what you thought?

**Comment by Gdsimms** [comment]
I have no answer. I have switched to using a few 'socat' commands to perform the serial sharing in my application.

**Comment by leoheck** [comment]
Ah, sorry I wanted to ask @danielinux , this MS GitHub is getting annoying. 

@Gdsimms this is interesting, I was using socat since it is more spread. But a coworker asked me to change to use this instead. I will give a socat another try to compare it with ttybus

**Comment by danielinux** [comment]
non-reliable simply means that there is no guarantee that all the bytes are transferred to the destination, because the output is non-blocking.

Imagine a tty transmitting to two devices attached. If one of the two is not 'ready to receive' (i.e. poll with POLLWR returns 0) only the other one will receive the payload.

Perhaps instead of 'non-reliable' it would have been more correct to use 'best-effort'.

**Comment by leoheck** [comment]
I see, thanks for your feedback @danielinux 

What about the data being transferred while both clients are connected and working. Can we say that both clients will always receive the same amount of data without losing any bytes?

I am splitting data from a gnss board connected to the USB into 2 instances of a server. It looks it works fine, but after a couple of hours or after a day something changes on one of the servers. I am not saying if ttybus is the issue, I just don't know where to look to find or debug this specific issue.

**Comment by OevreFlataeker** [comment]
> I have no answer. I have switched to using a few 'socat' commands to perform the serial sharing in my application.

Could you please describe how you used socat to multiplex the access to the ttyUSBx device?
